web/main/templates/includes/credits.html

```diff
@@ -45,23 +46,30 @@ <h4>Authors:</h4>
 
                             {% if row.incidental_authors %}
                             <br>
-                            <h4>Incorporating additional material from:</h4>
-                            <ul class="incidental-author-list">
-                                {% for user, clone in row.incidental_authors %}
-                                <li class="{% if user.verified_professor %} verified{% endif %}">
-                                    <a class="user" href="{% url 'dashboard' user.id %}">{{ user.display_name }}</a>
-                                    from 
-                                    {% if clone.is_public %}
-                                        <a href="{{ clone.get_absolute_url }}">{{ clone.title }}</a> 
-                                    {% else %}
-                                        {{ clone.title }}
-                                    {% endif %}
-                                    {% if clone.first_published.entry_date %}
-                                      (first published {{ clone.first_published.entry_date | date:"M Y" }}) 
-                                    {% endif %}
-                                </li>
+                            <h4 class="incidental-author-title"><em>{{ row.casebook.title }}</em> incorporates additional material from:</h4>
+
+                                {% for clone, authors in row.grouped_incidental_authors.items %}
+                                <ul class="incidental-author-list">
+                                    <li>
+
+                                        {% if clone.is_public %}
+                                            <a href="{{ clone.get_absolute_url }}">{{ clone.title }}</a> 
+                                        {% else %}
+                                            {{ clone.title }}
+                                        {% endif %}
+                                        {% if clone.first_published.entry_date %}
+                                            (first published {{ clone.first_published.entry_date | date:"M Y" }}) 
+                                        {% endif %}
+                                        with contributions from: 
+                                        {% for user in authors %}
+                                            <li class="user {% if user.verified_professor %} verified{% endif %}">
```

Comment:
I'd maybe wrap that `<li>` in a `<ul>` so you don't have 

```html
<ul>
  <li>
    <li></li>
  </li>
</ul>
```

My understanding is that in that situation, some HTML parsers will close both `<li>`s on the first `</li>` encountered, but results may vary.

**Example:**
<img width="596" alt="Screenshot 2023-05-02 at 11 42 54 AM" src="https://user-images.githubusercontent.com/625889/235716377-0c7ee063-55f5-4a81-b6cc-350ae4e7268f.png">

Reply:
Oh yeah good catch!